.github/workflows/default_on_push.yml

```diff
@@ -52,14 +56,16 @@ jobs:
           submodules: recursive
       - name: Prepare tags
         id: prep
-        run: |
-          DOCKER_IMAGE=${{ secrets.DOCKER_REPOSITORY }}
-          VERSION=latest
-          [[ $GITHUB_REF == refs/tags/* ]] && VERSION=${GITHUB_REF#refs/tags/v}
-          [[ $GITHUB_REF == 'refs/heads/stable' ]] && VERSION=stable
-          TAGS="${DOCKER_IMAGE}:${VERSION}"
-          [[ $VERSION =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]] && TAGS="$TAGS,${DOCKER_IMAGE}:latest"
-          echo ::set-output name=tags::${TAGS}
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: |
+            docker.io/mailserver/docker-mailserver
```

Comment:
One last thing to stickler on, if we can make these variables again - e.g.;

```yaml
images: |
  ${{ secrets.DOCKER_REPOSITORY }}
  ghcr.io/${{ github.repository_owner }}/docker-mailserver
```

Again, use-case being forks not having to change these values in the YAML. I don't know if the variable reference on that is correct (e.g., single or double `{` needed, but do test.

Reply:
> > ```
> > 
> > images: |
> > 
> >   docker.io/mailserver/docker-mailserver
> > 
> >   ghcr.io/docker-mailserver/docker-mailserver
> > ```
> > 
> > 
> > These values can of course again be transformed into github secrets.
> 
> I will create these secrets.

These secrets are now created:

```  INI
DOCKER_REPOSITORY = docker.io/mailserver/docker-mailserver
GHCR_REPOSITORY = ghcr.io/docker-mailserver/docker-mailserver
```

The old value from `DOCKER_REPOSITORY` just stays the same:) @wernerfred you can now adjust the lines to use both secrets.

Reply:
Resolved with https://github.com/docker-mailserver/docker-mailserver/pull/1763/commits/d6c2eed2f003f043f8f81e5d6374b28d25b6febf